src/views/blocks/attendees/order-links.php

```diff
@@ -62,5 +63,5 @@
 
 <div class="tribe-link-view-attendee">
 	<?php echo $message ?>
-	<a href="<?php echo esc_url( $link ) ?>"><?php echo sprintf( esc_html__( 'View your %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, $user_id, true ) ) ?></a>
+	<a href="<?php echo esc_url( $link ) ?>"><?php echo sprintf( esc_html__( 'View your %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, $user_id ) ) ?></a>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$user_id'

<sub>hash: 53d77eeb9413c13ebd6c78cf64917594</sub>